libraries/chain/include/graphene/chain/protocol/chain_parameters.hpp

```diff
@@ -30,6 +30,12 @@ namespace graphene { namespace chain { struct fee_schedule; } }
 
 namespace graphene { namespace chain {
 
+   struct committee_updatable_parameters
```

Comment:
If name it `committee_updatable_parameters` then it itself should be extendable, but currently it's not. (btw looks like the name is inspired by YOYOW)

An approach is to entirely replace `ext` with it and put optional fields inside, but then the struct with all fields are optional is not good for calls like `get_global_properties`.

So perhaps better to use a struct for objects to be stored in object database, and use another extendable struct for operations, and map the fields. ([YOYOW's approach](https://github.com/yoyow-org/yoyow-core/blob/v0.3.0-180626/libraries/chain/include/graphene/chain/protocol/committee_member.hpp#L164))

Reply:
@abitmore  These questions/comments will expose my ignorance. Sorry for the required mentoring, but here goes:

Your first paragraph seems to say that my naming is inconsistent, and inconsistent with Bitshares.  So if I change the name, does that negate your next two paragraphs?

Your second paragraph says (to me): "make the committee_updatable_parameters struct extendable. But that will make get_global_properties not work or be messy".

The proposed solution in paragraph 3 says (to me): "Look at how YOYOW did it. They use structs for persistence, and extendables for operations, and then map back and forth within operations."

Am I interpreting correctly?

Reply:
Since there is an `extension` field in the object, `get_global_properties` is still a bit messy. I guess I can live with that, although it's best if we have https://github.com/bitshares/bitshares-core/issues/167 done.

Reply:
Agreed that https://github.com/bitshares/bitshares-core/issues/167 would be nice. Perhaps I'll tackle that next.